released/packages/coq-itauto.8.13/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "frederic.besson@inria.fr"
+homepage: "https://gitlab.inria.fr/fbesson/itauto"
+dev-repo: "git@gitlab.inria.fr:fbesson/itauto.git"
+authors: ["Frédéric Besson"]
+bug-reports: "frederic.besson@inria.fr"
+license: "GPL 3"
+synopsis: "'itauto' is a reflexive SAT solver parameterised by a leaf tactic"
+build: [
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/Cdcl"]
+depends: [
+  "ocaml" {>= "4.9~"}
+  "dune" {>= "2.7.1"}
+  "coq" {>= "8.13.~" }
+  "ocamlbuild" {build }
```

Comment:
Wondering if `dune` and `ocamlbuild` are needed as dependencies, as I do not see them in the Makefile (but I probably miss something).

Reply:
There is a sub-makefile using `ocamlbuild`.
For `dune`, I don't remember. There used to be a dependency problem with coq. I'll check.
Thanks.